src/sign/ec_sign.c

```diff
@@ -76,10 +76,10 @@ CK_RV ec_main(CK_SESSION_HANDLE session) {
     CK_OBJECT_HANDLE privkey = CK_INVALID_HANDLE;
     rv = generate_ec_keypair(session, prime256k1, sizeof(prime256k1), &pubkey, &privkey);
     if (rv == CKR_OK) {
-        printf("secp256k1 key generated. Public key handle: %lu, Private key handle: %lu\n", pubkey,
+        printf("prime256k1 key generated. Public key handle: %lu, Private key handle: %lu\n", pubkey,
```

Comment:
This code became incorrect during the EL6 sign issue. We are using the prime256v1 curve, but the variable is  named prime256k1.

Could you refactor the variable to be prime256v1?

Reply:
Done